ci-operator/config/openshift/certman-operator/openshift-certman-operator-master.yaml

```diff
@@ -48,6 +48,23 @@ tests:
   commands: make validate
   container:
     from: src
+- as: leading-zero-check
+  postsubmit: true
+  steps:
+    test:
+    - as: check-commit-message
+      commands: |
+        COMMIT_MESSAGE=$(git log -1 --pretty=%B)
+        if [[ $COMMIT_MESSAGE =~ ^0 ]]; then
+            echo "Error: Commit message starts with a leading zero."
```

Comment:
```suggestion
            echo "Error: Git commit hash starts with a leading zero.  Please generate a new commit hash."
```

Reply:
You know...we _might_ be able to do this ourselves automatically.  There are automations in prow that generate new commits (eg: managed-cluster-config generating the final YAML and adding to the users' commits automatically).  We could theoretically create a "dummy" commit automatically and re-test for the leading zero.  That may help reduce overall toil.